### ISSUE TITLE
Commencer à parler écologie

### ISSUE THREAD

**Issue by laem** [laem/cartes#601]
Restaurant pas VG : avertissement vous risquez d'être déçu.

Stations essence.

**Comment by fouss** [comment]
Un petit filtre pour les commerces qui proposent du vrac aussi ça pourrait être très sympa 👌🏻

**Comment by laem** [comment]
Oui, pour ça ne pas hésiter à l'ajouter dans les catégories de recherche. 
Sur ce ticket, je veux explorer comment introduire l'écologie sans la chercher.
